service/src/main/resources/api/service_openapi.yaml

```diff
@@ -823,7 +920,23 @@ paths:
           $ref: '#/components/responses/NotFound'
         '500':
           $ref: '#/components/responses/ServerError'
-
+  /api/workspaces/v1/{workspaceId}/resources/referenced/gcp/bigquerydatatable/name/{name}:
```

Comment:
It looks like workspaces and references are plural and bigquerydatatable is singular. I  believe plural is the standard.